.github/workflows/test.yml

```diff
@@ -0,0 +1,43 @@
+name: Run tests
+
+on: 
+  pull_request:
+    branches:
```

Comment:
Can you clarify, will these test run if I'm on a different branch trying to merge into master?

Reply:
It will run on all pull requests into master or develop

Reply:
Could we change it to `*` and run on every PR to a branch. I believe that is the current behaviour with travis tests today